src/entity.hpp

```diff
@@ -46,9 +46,12 @@ class Entity
 	Sint32& char_energize;
 	Sint32& char_torchtime;
 	Sint32& char_poison;
+	Sint32 char_fire; // Counter for how many ticks Entity will be on fire
```

Comment:
This should imitate the poison code; an Sint32& reference to a skill[].

Reply:
Apologies, I had forgotten to make that change, as I had received a response from WALL. Will be fixed momentarily.